### PR TITLE
Upgrade toml-edit crate

### PR DESCRIPTION
**Pull request by ssrlive**

### CODE REVIEW

**Comment by ssrlive** [comment]
Hi @jplatte , I have upgraded the toml-edit crate, please review it.

**Comment by jplatte** [comment]
Hi, could you please make the CI changes their own separate PR and write some words on the _why_ of that change (either as part of the commit message, or the PR description)?

File: rustfmt.toml
Comment (jplatte) [inline rustfmt.toml:1]: Also, why did you delete a bunch of settings from the rustfmt config?

Comment (ssrlive) [reply rustfmt.toml:1]: because the deleted items can not pass the nightly CI.

**Comment by ssrlive** [comment]
I noticed that when you released the version, it failed. I checked the usage of the components used by the script and modified them accordingly.

> Hi, could you please make the CI changes their own separate PR and write some words on the _why_ of that change (either as part of the commit message, or the PR description)?

**Comment by ssrlive** [comment]
I have test it just now. It's can release the packages correctly. I have so many other jobs. I hope you won't make things difficult for me.

image

**Comment by jplatte** [comment]
There's still way too many changes mixed in this one PR. This is a pretty low-priority project to me, otherwise I would have split the PR up myself probably.

**Comment by ikrivosheev** [comment]
@jplatte I started splitting this MR into parts. This is first: https://github.com/DevinR528/cargo-sort/pull/79

**Comment by ikrivosheev** [comment]
And finished PR: https://github.com/DevinR528/cargo-sort/pull/81
